test/configpaths.cpp

```diff
@@ -13,19 +13,8 @@
 
 using namespace newsboat;
 
-inline namespace {
-int mkdir(const Filepath& dirpath, mode_t mode)
-{
-	const auto dirpath_str = dirpath.to_locale_string();
-	return ::mkdir(dirpath_str.c_str(), mode);
-}
-
-bool file_available_for_reading(const Filepath& filepath)
-{
-	const auto filepath_str = filepath.to_locale_string();
-	return (0 == ::access(filepath_str.c_str(), R_OK));
-}
-}
+int mkdir(const Filepath& dirpath, mode_t mode);
+bool file_available_for_reading(const Filepath& filepath);
```

Comment:
Ah, I see these functions are copied all over the place. Let's move them to test_helpers/misc.{h,cpp} instead.